src/airflow/airqo_etl_utils/bigquery_api.py

```diff
@@ -19,6 +19,9 @@ def __init__(self):
         self.analytics_table = configuration.BIGQUERY_ANALYTICS_TABLE
         self.sites_table = configuration.BIGQUERY_SITES_TABLE
         self.devices_table = configuration.BIGQUERY_DEVICES_TABLE
+        self.calibrated_hourly_measurements_table = (
+            configuration.BIGQUERY_CALIBRATED_HOURLY_EVENTS_TABLE
+        )
```

Comment:
table to temporary stored the historical calibrated data for analysis and verification before updating the main tables.